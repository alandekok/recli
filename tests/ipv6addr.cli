ipv6 ::1
ipv6 11:22
ipv6 -3.3
ipv6 xyz
