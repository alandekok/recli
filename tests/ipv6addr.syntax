ipv6 IPV6ADDR
