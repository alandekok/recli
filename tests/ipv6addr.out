ipv6 -3.3
     ^ Invalid character in IPv6 address.
ipv6 xyz
     ^ Invalid character in IPv6 address.
